dub.sdl

```diff
@@ -3,3 +3,4 @@ description `A "tagged union" implementation with transparent operator forwardin
 authors "Sönke Ludwig"
 copyright "Copyright © 2015, Sönke Ludiwg"
 license "BSL-1.0"
+dflags "-preview=dip1000"
```

Comment:
I'm actually not quite sure how this is handled when taggedalgebraic gets compiled into the same static library as another package (e.g. using `dub build --combined`), at least it won't get inherited, but it feels cleaner to just add this to the unittest build type:
```
buildType "unittest" {
    buildOptions "unittests" "debugMode" "debugInfo"
    dflags "-preview=dip1000"
}
```